spec/components/post_revisor_spec.rb

```diff
@@ -596,6 +596,12 @@
         expect(post.topic.word_count).to eq(5)
       end
 
+      it 'increases the post_edits stat count' do
+        expect(coding_horror.user_stat.post_edits_count).to eq(1)
+        subject.revise!(post.user, raw: "This is a new revision")
+        expect(coding_horror.user_stat.post_edits_count).to eq(2)
```

Comment:
```suggestion
        expect { subject.revise!(post.user, raw: "This is a new revision") }.to change { coding_horror.user_stat.reload.post_edits_count }.by(1)
```

Instead of repeating the same assertion twice, there is a nice RSpec matcher we can use here.